updater/lib/dependabot/service.rb

```diff
@@ -64,9 +49,7 @@ def wait_for_calls_to_finish
 
     sig { params(dependency_change: Dependabot::DependencyChange, base_commit_sha: String).void }
     def create_pull_request(dependency_change, base_commit_sha)
-      if Experiments.enabled?("dependency_change_validation")
-        check_dependencies_have_previous_version(dependency_change.updated_dependencies)
-      end
+      dependency_change.check_dependencies_have_previous_version if Experiments.enabled?("dependency_change_validation")
```

Comment:
I assume this will still throw the exception and I think the plan was to swallow the error, log it silently and continue?

Reply:
Yes, but I am leaving it in for now to verify that what I am changing is effective. If it was the exception will stop happening.